transport/tchannel/outbound_test.go

```diff
@@ -136,6 +137,7 @@ func TestCallSuccess(t *testing.T) {
 				0x00, 0x0b, 'f', 'o', 'o', '-', 'B', 'A', 'R', '-', 'B', 'a', 'Z',
 				0x00, 0x09, 'F', 'o', 'o', 'B', 'a', 'r', 'B', 'a', 'z',
 			},
+			addSvcName: false,
```

Comment:
can drop the `false`s and rely on go's defaults

Reply:
Resolved.